ompi/mpi/fortran/use-mpi-f08/Makefile.prototype_files

```diff
@@ -0,0 +1,171 @@
+        $(abs_top_srcdir)/ompi/mpi/fortran/use-mpi-f08/recv_init_ts.c.in \
+        $(abs_top_srcdir)/ompi/mpi/fortran/use-mpi-f08/reduce_local_ts.c.in \
+        $(abs_top_srcdir)/ompi/mpi/fortran/use-mpi-f08/reduce_scatter_block_ts.c.in \
+        $(abs_top_srcdir)/ompi/mpi/fortran/use-mpi-f08/reduce_scatter_block_init_ts.c.in \
+        $(abs_top_srcdir)/ompi/mpi/fortran/use-mpi-f08/reduce_scatter_ts.c.in \
+        $(abs_top_srcdir)/ompi/mpi/fortran/use-mpi-f08/reduce_scatter_init_ts.c.in \
+        $(abs_top_srcdir)/ompi/mpi/fortran/use-mpi-f08/rget_accumulate_ts.c.in \
+        $(abs_top_srcdir)/ompi/mpi/fortran/use-mpi-f08/rget_ts.c.in \
+        $(abs_top_srcdir)/ompi/mpi/fortran/use-mpi-f08/rput_ts.c.in \
+        $(abs_top_srcdir)/ompi/mpi/fortran/use-mpi-f08/rsend_init_ts.c.in \
+        $(abs_top_srcdir)/ompi/mpi/fortran/use-mpi-f08/rsend_ts.c.in \
+        $(abs_top_srcdir)/ompi/mpi/fortran/use-mpi-f08/scan_ts.c.in \
+        $(abs_top_srcdir)/ompi/mpi/fortran/use-mpi-f08/scan_init_ts.c.in \
+        $(abs_top_srcdir)/ompi/mpi/fortran/use-mpi-f08/scatter_ts.c.in \
+        $(abs_top_srcdir)/ompi/mpi/fortran/use-mpi-f08/scatter_init_ts.c.in \
+        $(abs_top_srcdir)/ompi/mpi/fortran/use-mpi-f08/scatterv_ts.c.in \
+        $(abs_top_srcdir)/ompi/mpi/fortran/use-mpi-f08/scatterv_init_ts.c.in \
+        $(abs_top_srcdir)/ompi/mpi/fortran/use-mpi-f08/send_init_ts.c.in \
+        $(abs_top_srcdir)/ompi/mpi/fortran/use-mpi-f08/sendrecv_replace_ts.c.in \
+        $(abs_top_srcdir)/ompi/mpi/fortran/use-mpi-f08/sendrecv_ts.c.in \
+        $(abs_top_srcdir)/ompi/mpi/fortran/use-mpi-f08/ssend_init_ts.c.in \
+        $(abs_top_srcdir)/ompi/mpi/fortran/use-mpi-f08/ssend_ts.c.in \
+        $(abs_top_srcdir)/ompi/mpi/fortran/use-mpi-f08/type_vector.c.in \
+        $(abs_top_srcdir)/ompi/mpi/fortran/use-mpi-f08/type_size.c.in \
+        $(abs_top_srcdir)/ompi/mpi/fortran/use-mpi-f08/type_indexed.c.in \
+        $(abs_top_srcdir)/ompi/mpi/fortran/use-mpi-f08/type_contiguous.c.in \
+        $(abs_top_srcdir)/ompi/mpi/fortran/use-mpi-f08/type_create_darray.c.in \
+        $(abs_top_srcdir)/ompi/mpi/fortran/use-mpi-f08/type_get_true_extent.c.in \
+        $(abs_top_srcdir)/ompi/mpi/fortran/use-mpi-f08/type_create_subarray.c.in \
+        $(abs_top_srcdir)/ompi/mpi/fortran/use-mpi-f08/type_create_struct.c.in \
+        $(abs_top_srcdir)/ompi/mpi/fortran/use-mpi-f08/type_create_hindexed.c.in \
+        $(abs_top_srcdir)/ompi/mpi/fortran/use-mpi-f08/type_create_hindexed_block.c.in \
+        $(abs_top_srcdir)/ompi/mpi/fortran/use-mpi-f08/type_create_indexed_block.c.in \
+        $(abs_top_srcdir)/ompi/mpi/fortran/use-mpi-f08/type_create_hvector.c.in \
+        $(abs_top_srcdir)/ompi/mpi/fortran/use-mpi-f08/psend_init_ts.c.in \
+        $(abs_top_srcdir)/ompi/mpi/fortran/use-mpi-f08/precv_init_ts.c.in
+
+extra_dist_prototype_files =$(subst $(abs_top_srcdir)/ompi/mpi/fortran/use-mpi-f08/,,$(prototype_files))
```

Comment:
Does this work in BSD Make?

Reply:
I don't know.  is there an equivalent or near equivalent for subst that we could use here?

Reply:
i'll try that approach you mentioned above but I thought that didn't work when I first tried it.